jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TooltipSection.java

```diff
@@ -0,0 +1,71 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.common.Checkbox;
+import com.epam.jdi.light.angular.elements.common.Tooltip;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.TextField;
+
+public class TooltipSection extends Section {
+
+    /* Basic tooltip */
```

Comment:
The comments here indicates that there is a problem with code with they describe.